--- lalapps-10.0.0/Makefile.am.orig	2024-06-09 22:17:54.694541384 +0200
+++ lalapps-10.0.0/Makefile.am	2024-06-10 06:24:28.846383094 +0200
@@ -22,7 +22,4 @@ user_environment = \
 	trailingcolon MANPATH; \
 	set $(PACKAGE)_PREFIX $(prefix); \
 	set $(PACKAGE)_DATADIR $(pkgdatadir); \
-	prepend MANPATH $(mandir); \
-	prepend PATH $(bindir); \
-	prepend PYTHONPATH $(pyexecdir) $(pythondir); \
 	$(END_OF_LIST)
